requirements/dev.txt

```diff
@@ -14,8 +14,6 @@ charset-normalizer==3.4.3
     # via
     #   reportlab
     #   requests
-colorama==0.4.6
```

Comment:
The removals in requirements files is because those were added by compiling dependencies on windows. When compiling them again on a UNIX system, they got removed again.